pennylane/ops/qutrit/observables.py

```diff
@@ -133,3 +146,136 @@ def compute_diagonalizing_gates(eigenvectors, wires):  # pylint: disable=argumen
 
         """
         return [QutritUnitary(eigenvectors.conj().T, wires=wires)]
+
+
+class GellMannObs(Observable):
+    r"""
+    The Gell-Mann observables for qutrits
+
+    The Gell-Mann matrices are a set of 8 linearly independent :math:`3x3` traceless, Hermitian matrices which
+    naturally generalize the Pauli matrices from :math:`SU(2)` to :math:`SU(3)`.
+
+    .. math::
```

Comment:
The rendering here still looks a little odd; maybe try using an `align` or table environment?

![image](https://user-images.githubusercontent.com/2068515/189457401-3c532e6d-52e3-45ee-bf9f-eee50ff4876e.png)

Reply:
I fixed the alignment issue. Let me know if it looks better now.

Reply:
Beautiful :art: